src/DIRAC/Core/Utilities/DIRACScript.py

```diff
@@ -45,7 +48,7 @@ def __call__(self, func=None):
     matches = defaultdict(list)
     function_name = None
     for entrypoint in metadata.entry_points()['console_scripts']:
-      if not entrypoint.name.startswith("dirac-"):
+      if not entrypoint.name.startswith("dirac-") or self.scriptName != entrypoint.name:
```

Comment:
I don't think it's a problem but out of curiosity, what's the reason for this?

Reply:
you are right, in the context of this PR it makes no sense. I'll take it off. thanks